### DISCUSSION TITLE
TQL25-2211 Zemismart Raex Cover / Roller Shade Motor -- anyone have it working?

### DISCUSSION THREAD

**Discussion by Ltek** [Koenkk/zigbee2mqtt#18782] [Devices]
Anyone have it working?  Looks identical to the photo in this supported device... https://www.zigbee2mqtt.io/devices/ZM25RX-08_30.html

When I pair I get...
Info | Successfully interviewed '0x30fb10fffe75280a', device has successfully been paired
Warning | Device '0x30fb10fffe75280a' with Zigbee model 'TQL25-2211' and manufacturer name 'Raex' is NOT supported, please follow https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

Amazon Alexa pairs with it and identifies as "Raex Smart Device" and correctly shows it as a "Blind"

label on the motor is: TQL25-SBN
7.4vdc, 2200mAH, 35rpm
https://www.raexaustralia.com/product/electronic-wireless-roller-blind-motor-with-mounting-kit/#:~:text=Description,one%20motor%20for%20every%20scenario.

I'm 100% new to Z2M but have used Deconz for a long time. Happy to test if someone has a config file. thx!

**Comment by MattDemers** [comment]
Sorry to necro this, but it looks like I got the same device. Did you ever get this working?

**Reply by Ltek** [reply]
nope, returned it and using the TuYa TS0601 (EL25)
